src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -7063,7 +7085,7 @@ private BlockSyntax ParseBlock(bool isMethodBody = false, bool isAccessorBody =
 
             // There's a special error code for a missing token after an accessor keyword
             var openBrace = isAccessorBody && this.CurrentToken.Kind != SyntaxKind.OpenBraceToken
-                ? this.AddError(SyntaxFactory.MissingToken(SyntaxKind.OpenBraceToken), ErrorCode.ERR_SemiOrLBraceExpected)
+                ? this.AddError(SyntaxFactory.MissingToken(SyntaxKind.OpenBraceToken), ErrorCode.ERR_SemiOrLBraceOrArrowExpected)
```

Comment:
Do we still need this special error for accessors? Can we simply do what we are doing for regular methods instead? In any case, I think this code should be conditioned of whether the feature of expression-bodied accessors is enabled.